packages/core-data/src/hooks/use-entity-record.ts

```diff
@@ -182,7 +182,10 @@ export default function useEntityRecord< RecordType >(
 	const { data: record, ...querySelectRest } = useQuerySelect(
 		( query ) => {
 			if ( ! options.enabled ) {
-				return null;
+				return {
+					// Avoiding returning a new reference on every execution.
```

Comment:
There's no need for this comment; The `null` value, like other primitive values, always has a stable reference.

Reply:
@Mamaduka 

Thanks for the review! Now I would like to delete this comment before merging.

Reply:
Thanks, @t-hamano!